snooty.toml

```
@@ -12,7 +12,7 @@ intersphinx = [
 ]
 
 [constants]
-docs-branch = "master" # always set this to the docs branch (i.e. master, 1.7, 1.8, etc.)
-version = "v1.8.0" # always set this to the driver branch (i.e. v1.7.0, v1.8.0, etc.)
+doc-branch = "master" # always set this to the docs branch (i.e. master, 1.7, 1.8, etc.)
```

Issue: I think this is causing an error in the build log and making Usage Example code Github links 404. It is referenced on line 17 below.

```suggestion
docs-branch = "master" # always set this to the docs branch (i.e. master, 1.7, 1.8, etc.)
```

Builde log: "ERROR(snooty.toml:0ish): docs-branch not defined as a source constant"